developer/src/kmc-ldml/test/test-tran.ts

```diff
@@ -116,24 +116,42 @@ describe('tran', function () {
       }
     },
     {
+      // a more complicated reorder group, from the spec
       subpath: 'sections/ordr/nod-Lana.xml',
       callback(sect) {
         const tran = <Tran> sect;
         assert.equal(tran.groups?.length, 1);
         assert.equal(tran.groups[0].type, constants.tran_group_type_reorder);
-        // const { reorders } = tran.groups[0];
-        // console.dir({reorders}, {depth: Infinity, color: true});
-        // assert.lengthOf(reorders, 1);
-        // assert.lengthOf(reorders[0].elements, 4);
-        // assert.strictEqual(reorders[0].elements[0].value.value, "ខ");
-        // assert.strictEqual(reorders[0].elements[1].value.value, "ែ");
-        // assert.strictEqual(reorders[0].elements[2].value.value, "្");
-        // assert.strictEqual(reorders[0].elements[3].value.value, "ម");
-        // assert.strictEqual(reorders[0].elements[0].order, 1);
-        // assert.strictEqual(reorders[0].elements[1].order, 3);
-        // assert.strictEqual(reorders[0].elements[2].order, 4);
-        // assert.strictEqual(reorders[0].elements[3].order, 2);
-        // assert.isEmpty(reorders[0].before);
+        const { reorders } = tran.groups[0];
+        assert.lengthOf(reorders, 6);
```

Comment:
This doesn't seem related to the rest of the PR, at least on the face of things.  Was this test-spec dependent on these changes?  Why is it being altered / extended now?

I do see a few `assert.sameDeepOrderedMembers` checks below, but none of them has a compound range like `[[0x41, 0x7A], [0x0127, 0x0127]]` - I only see simple, single-interval ranges.

Reply:
Parsing the reorders is dependent on uset. But the final test changes are in a different PR. These shouldn't have been included.